tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -57,6 +58,7 @@ function install_artifacts() {
 	echo "copying kata artifacts onto host"
 	cp -a /opt/kata-artifacts/opt/kata/* /opt/kata/
 	chmod +x /opt/kata/bin/*
+	chmod +x /opt/kata/runtime-rs/bin/*
```

Comment:
Usually it should be `/opt/kata/bin/runtime-rs/`

Reply:
As `containerd-shim-kata-v2` is installed under this path, and it is an executable file for runtime-rs. I think it's better to put it under `bin`